test/e2e/types.go

```diff
@@ -47,13 +49,20 @@ type VerleroConfig struct {
 	ResticHelperImage        string
 	UpgradeFromVeleroVersion string
 	UpgradeFromVeleroCLI     string
+	MigrateFromVeleroVersion string
+	MigrateFromVeleroCLI     string
 	Plugins                  string
 	AddBSLPlugins            string
 	InstallVelero            bool
 	KibishiiDirectory        string
 	Features                 string
 	Debug                    bool
 	GCFrequency              string
+	DefaultCluster           string
+	StandbyCluster           string
+	ClientToBeInstall        TestClient
+	DefaultClient            TestClient
+	StandbyClient            TestClient
```

Comment:
Maybe it is better to use pointer type (*TestClient)

Reply:
Right, good point!